internal/migration/migration.go

```diff
@@ -0,0 +1,478 @@
+package migration
+
+import (
+	"context"
+	"encoding/binary"
+	"errors"
+	"fmt"
+	"os"
+	"time"
+
+	json "github.com/nikkolasg/hexjson"
+	"go.etcd.io/bbolt"
+
+	"github.com/drand/drand/chain"
+	"github.com/drand/drand/chain/postgresdb/database"
+	"github.com/drand/drand/chain/postgresdb/pgdb"
+	"github.com/drand/drand/chain/postgresdb/schema"
+	"github.com/drand/drand/log"
+)
+
+type (
```

Comment:
Nit Pick, can't these have the keyword type attached instead of being in this group?